excessive_rolling_update.yml

```diff
@@ -54,8 +54,58 @@
         - current_version.stdout_lines[0].replace(' ','').split(':')[1] < "v2.0.1"
         - tidb_version >= "v2.1.0" or tidb_version == "latest"
 
+- hosts: pd_servers
+  any_errors_fatal: true
+  serial: 1
+  tags:
+    - pd
+  tasks:
+    - name: Check pd cluster status
+      uri:
+        url: "http://{{ ansible_host }}:{{ pd_client_port }}/pd/health"
+        method: GET
+        return_content: yes
+        status_code: 200
+      register: pd_status
+    - name: Failed when one node of pd is unhealthy
+      fail:
+        msg: "Some pd node is unhealth"
+      when: "'false' in pd_status.content"
+
+- hosts: pd_servers
+  any_errors_fatal: true
+  serial: 1
+  tags:
+    - pd
+  tasks:
+    - name: Get pd leader
+      uri:
+        url: "http://{{ ansible_host }}:{{ pd_client_port }}/pd/api/v1/leader"
+        method: GET
+        return_content: yes
+        status_code: 200
+      register: pd_leader_info
+    - set_fact:
+        pd_leader: "{{ pd_leader_info.json | json_query('client_urls') }}"
+    - name: Set pd follower list
+      add_host:
+        name: "{{ ansible_host }}"
+        groups: pd_servers_followers
+        pd_client_port: "{{ pd_client_port }}"
+        deploy_dir: "{{ deploy_dir }}"
+        pd_peer_port: "{{ pd_peer_port }}"
```

Comment:
It seems that pd_servers_followers groups don't have group variables in  
https://github.com/pingcap/tidb-ansible/blob/master/group_vars/pd_servers.yml and other customized  host variables.